vowpalwabbit/core/src/reductions/cb/cb_explore_adf_large_action_space.cc

```diff
@@ -258,15 +258,15 @@ void generate_Z(const multi_ex& examples, Eigen::MatrixXf& Z, Eigen::MatrixXf& B
 template <typename T, typename S>
 cb_explore_adf_large_action_space<T, S>::cb_explore_adf_large_action_space(uint64_t d, float gamma_scale,
     float gamma_exponent, float c, bool apply_shrink_factor, VW::workspace* all, uint64_t seed, size_t total_size,
-    size_t thread_pool_size, size_t block_size, implementation_type impl_type)
+    size_t thread_pool_size, size_t block_size, bool use_explicit_simd, implementation_type impl_type)
```

Comment:
is there a reason we are enabling this by both turning on a compile time flag and a cli argument?

Reply:
just saw the remark in the description of the PR